src/System.Windows.Forms/src/System/Windows/Forms/ListBox.SelectedObjectCollection.cs

```diff
@@ -283,7 +279,7 @@ public void Remove(object value)
                     ObjectCollection items = _owner.Items;
                     if (items is not null & value is not null)
```

Comment:
This looks like a typo, which is flagged by the analyzer with the NRTA
```suggestion
                    if (items is not null && value is not null)
```